spec/decorators/subscription_decorator_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe SubscriptionDecorator do
+  include Rails.application.routes.url_helpers
```

Comment:
I believe you have access to url helpers using `h.` without needing to explicitly include the helpers.
So if you need to access 'projects_url' you can call `h.projects_url`.